.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I'm not sure if this is going to work on windows machines.

Reply:
I'm on a windows machine and don't have any problem with it. If people run into an issue we can look into changing it.

typically if you can run git you can run bash scripts as well.